src/styles/font-jrkf/iconfont.js

```
@@ -0,0 +1,68 @@
+        m,
+        n = function (a, l) {
+          l.parentNode.insertBefore(a, l)
+        }
+      if (l && !o.__iconfont__svg__cssinject__) {
+        o.__iconfont__svg__cssinject__ = !0
+        try {
+          document.write(
+            '<style>.svgfont {display: inline-block;width: 1em;height: 1em;fill: currentColor;vertical-align: -0.1em;font-size:16px;}</style>'
+          )
+        } catch (a) {
+          console && console.log(a)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)